content/influxdb/v2.0/tools/clis.md

```diff
@@ -0,0 +1,28 @@
+---
+title: InfluxDB CLIs
+description:
+  Use the `influx` and `indfluxd` command line interfaces to interact with and
+  manage InfluxDB.
+menu:
+  influxdb_2_0:
+    parent: Tools & integrations
+weight: 102
+influxdb/v2.0/tags: [cli]
+---
+
+Use the `influx` and `indfluxd` command line interfaces (CLIs) to interact with and
+manage InfluxDB.
+
+## influx CLI
+Use the `influx` CLI to interact with and manage both **InfluxDB Cloud** and **InfluxDB OSS**.
+Write and query data, generate InfluxDB templates, export data, manage organizations
+and users, and more.
+
+_For more information, see the [`influx` reference documentation](/influxdb/v2.0/reference/cli/influx/)._
+
+## influxd CLI
```

Comment:
```
## `influxd` CLI
```

Reply:
We override/don't allow `<code>` styling in headings, so this won't render any differently. Adding the inline code will just add unnecessary html markup.